src/UseCases/office.js

```diff
@@ -0,0 +1,37 @@
+const Office = require('../Models/office');
+const bcrypt = require('../Lib/bcrypt');
+
+function getOffices(){
+    return Office.find({})
+}
+
+function getById(idOffice){
+    return Office.findById(idOffice)
+}
+
+function create(dataOffice){
+    const {name, address, email, password, rfc, clients, residents, projects} = dataOffice
+
+    const emailFound = await Office.findOne({office: email})
```

Comment:
```suggestion
    const emailFound = await Office.findOne({email: email})
```

Para que busque por el email abusados con esta parte. @KristomeisterDev

Reply:
estaba seguro que era office: email alo mejor como estoy en clase zoom me consume mucho y luego me salen errores

Reply:
ah, ya vi perdón le entendí mal.

Reply:
ya quedo Fer